### DISCUSSION TITLE
Access MS Project Online

### DISCUSSION THREAD

**Discussion by a-shawky** [pnp/pnpcore#1544] [Q&A] [answered]
Hello,

I am trying to develop a solution with pnp core to get MS projects online hosted under SharePoint to publish the projects that are not published.

 I successfully setup the library  and able to query the SharePoint site that has the projects, but I can not find a list with name projects.

Could anyone help me please?

**Comment by jansenbe** [comment, marked as answer]
@a-shawky : PnP Core SDK does not have specific Project Online support, think you need their APIs to do what you're asking for.
